kombu/asynchronous/timer.py

```diff
@@ -13,23 +13,24 @@
 
 from kombu.log import get_logger
 
-try:
```

Comment:
No change necessary, but to save us the conditional handling of `ZoneInfo`, as we only want UTC we could use Python's `datetime.timezone.utc`.

Reply:
I am +1 on this, seems like a less risky change than introducing `zoneinfo`